### ISSUE TITLE
Sending zero-length messages causes exception

### ISSUE THREAD

**Issue by datacoda** [valotrading/zeromq-scala-binding#4]
Send(byte[] msg, ... ) doesn't handle zero-length frames and throws an exception when calling newZmqMessage(byte[]...) with a zero length array.  I did a quick fix locally by at least allocating one byte for the memory buffer, but that's a bit hokey.

**Comment by kro** [comment]
Thanks for letting me know. I'll fix that tomorrow.

**Comment by kro** [comment]
I assume that this issue is related to the ZeroMQ module for Akka, right? I am unable to reproduce the issue, see:

https://github.com/jboner/akka/commit/a911a1ffe5f863e052807d8d8866b3b21369c5fe

**Comment by datacoda** [comment]
Thanks kro,

It looks to be unrelated to the akka commit.

I was messing around with creating message envelopes for a router-based custom device.  Hence the need to call send(..) directly.

``` scala
socket.send(address.getBytes, ZMQ.SNDMORE)
socket.send("".getBytes, ZMQ.SNDMORE)  // throws exception here
socket.send(message.getBytes, 0)
```

will throw the allocation must be > 0 exception.

**Comment by datacoda** [comment]
Sorry, wrong button.  _sigh_

**Comment by kro** [comment]
Thanks dataferret for the code snippet. The issue is now fixed in version 0.0.2-SNAPSHOT. Let me know if you come across some other bugs :)
